tests/neg/outdent-dot.scala

```diff
@@ -0,0 +1,13 @@
+def Block(f: => Int): Int = f
+
+def bar(): String =
+  Block:
+      2 + 2
+     .toString  // error
```

Comment:
I worry that this might be confusing for users, since the rules for this will be quite hard to find and a difference of one whitespace might cause hard to understand errors.

I would argue that indentation for `.toString` could be anything between the indentation of `Block` and the indentation of `2 + 2`.

The alternative for this:
```
Block{
      2 + 2
     }.toString
```
works and dropping braces will break it.

Why does `.toString` need to match the indentation of anything before? It's more a continuation of the previous statement and not a standalone statement itself. We are already pretty lenient for a similar syntax with multiple parameter clauses.

Reply:
The one-space rule is there to not open the flood-gates that small changes can have a silent change in meaning.